### DISCUSSION TITLE
Should I commit schemas?

### DISCUSSION THREAD

**Discussion by martpie** [tauri-apps/tauri#9051] [Q&A] [answered]
Tauri 2.0 generates a bunch of files in `src-tauri/gen/schemas`. I code between macOS and Windows, and running `tauri dev` on each platform generates different json, meaning I'm always committing an updates version of those files when changing OS.

Should I commit or exclude those from my version control?

**Comment by amrbashir** [comment, marked as answer]
these should be excluded and ignored in git, they will be generated on each build if needed
